src/components/Icon/AngryEmoji.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+type Props = {
+  className?: string
+  size?: number
+}
+
+function AngryEmoji({ className, size }: Props) {
+  return (
+    <svg
+      width={size || '24'}
+      height={size || '24'}
+      className={className}
+      viewBox="0 0 23 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path d="M0 23.61H23V0.61H0V23.61Z" fill="url(#pattern2)" />
+      <defs>
+        <pattern id="pattern2" patternContentUnits="objectBoundingBox" width="1" height="1">
+          <use xlinkHref="#image0_7148_2825" transform="scale(0.00625)" />
+        </pattern>
+        <image
```

Comment:
As this use an image... does it make sense to use an svg? Can't we use a real emoji?